bsip-0062.md

```diff
@@ -0,0 +1,44 @@
+ ```
+  BSIP: 0062
+  Title: Close short position 
+  Authors:
+    Stefan Schießl stefan.schiessl@blockchainprojectsbv.com
+    Jerry Liu bitcrab@qq.com
+  Status: Draft
+  Type: Protocol
+  Created: 2019-09-14
+  Worker: TBD
+  ```
+
+# Abstract
+Closing a short position comes with a risk of getting margin called if no spare funds are available to buy the long and reduce the debt to zero. This BSIP introduces a way to put the short position directly on the market.
+
+# Rationale
+Closing a short position with a healthy CR should come with no risk of getting margin called. Currently the user has to take out collateral from the short position in order to put it on the market. Reducing the collateral increases the risk of margin call. This is especially cumbersome and risky if you want to close a position that has CR close to MCR (in that case you can only take out a small amount of collateral, sell it, reduce debt, repeat until position closed).
```

Comment:
> Currently the user has to take out collateral from the short position in order to put it on the market.

This is not generally true.  It would be better to qualify this statement with something like the phrase that is used in the abstract.

"Currently a user, **who has no spare funds**, has to take out collateral from the short position in order to put it on the market."